### ISSUE TITLE
05-app-tags_middle.t fails (if Tags::HTML::Page::Begin is too old?)

### ISSUE THREAD

**Issue by eserte** [michal-josef-spacek/Plack-Component-Tags-HTML#1]
On some of my smoker systems the test suite fails:
```
#   Failed test 'Content (real HTML content).'
#   at t/Plack-Component-Tags-HTML/05-app-tags_middle.t line 40.
#          got: '<!DOCTYPE html>
# <html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><title>My app</title><style type="text/css">
# </style></head><body>Hello world</body></html>'
#     expected: '<!DOCTYPE html>
# <html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><title>My app</title></head><body>Hello world</body></html>'
# Looks like you failed 1 test of 4.
t/Plack-Component-Tags-HTML/05-app-tags_middle.t .. 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/4 subtests 
```

Statistical analysis suggests that the tests passes only if Tags::HTML::Page::Begin 0.11 or 0.12 is installed:
```
****************************************************************
Regression 'mod:Tags::HTML::Page::Begin'
****************************************************************
Name           	       Theta	      StdErr	 T-stat
[0='const']    	      0.0000	      0.0000	   5.52
[1='eq_0.11']  	      1.0000	      0.0000	17571546824755152.00
[2='eq_0.12']  	      1.0000	      0.0000	22027099659802804.00

R^2= 1.000, N= 85, K= 3
****************************************************************
```

**Comment by michal-josef-spacek** [comment]
@eserte Thanks, you are right. I am fixing this.

**Comment by michal-josef-spacek** [comment]
Fixed in 763c6307d67983b0f6df58da94622f7a05ddf093
